orangecontrib/network/readwrite.py

```diff
@@ -295,7 +300,8 @@ def read_pajek(path, encoding='UTF-8', project=False, auto_table=False):
     for node in G.node:
         G.node[node]['label'] = node
     nx.relabel_nodes(G, remapping, copy=False)
-    assert not table or len(table) == G.number_of_nodes(), 'There was a bug in NetworkX. Please update to git if need be'
+    if not table or len(table) == G.number_of_nodes():
+        raise PajekBug("There was a bug in NetworkX. Please update to git if need be")
```

Comment:
"There is a bug in your version of NetworkX reading Pajek files. Please update your NetworkX installation."